pkgs/development/libraries/gourou/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, stdenv, fetchgit, fetchurl
+, pugixml, updfparser, curl, openssl, libzip }:
+
+let
+  base64 = fetchurl {
+    url = "https://gist.githubusercontent.com/tomykaira/f0fd86b6c73063283afe550bc5d77594/raw/7d5a89229a525452e37504976a73c35fbaf2fe4d/Base64.h";
+    sha256 = "sha256-BKK+6mPEEEd4Y8c6GU2X6UB5fQ+3nPbUbEF8zQ6nGM4=";
+  };
+in stdenv.mkDerivation rec {
+  name = "gourou";
+  version = "0.7.2";
+
+  src = fetchgit {
+    url = "git://soutade.fr/libgourou";
+    rev = "v${version}";
+    sha256 = "sha256-jvvfrCuRggSPk7p/WtEaKRV7kILuZjUB1KEgaE4LStc=";
+  };
+
+  postUnpack = ''
```

Comment:
I know I said post unpack, but I meant post unpack phase for any phase post unpack, not necessary the `postUnpack` phase.

Actually this looks more like to be a `postPatch` or `preBuild` phases, instead of `postUnpack` (since this isn't unpacking anything).